### DISCUSSION TITLE
not getting realtime notifications

### DISCUSSION THREAD

**Discussion by TD-4242** [magicbell/community#179] [Questions] [answered]
started getting cors issues talking to ably again.

![image](https://github.com/magicbell-io/community/assets/3487671/0db550ef-cb21-42bc-93e5-8a1e5b0bfc1c)

This was working yesterday, but today all environments are responding with this.

**Comment by unamashana** [comment]
Sorry about that @TD-4242. This seems to be an issue over at Ably. While we write to them, can you please confirm if you are not receiving real-time updates? Ably's SDK can be quite verbose and often has errors that don't cause service degradation.

**Reply by TD-4242** [reply]
yes, I first noticed I was not getting real time notifications and started looking for errors.

**Reply by unamashana** [reply]
We've reached out to Ably to consult with them on this issue. We will keep you posted.

**Reply by smeijer** [reply]
While we wait for ably, I have a few questions if you don't mind. 

- is this on a corporate/public network, or at home?
- any chance that a proxy is removing headers?
- do you have the same issue on another device? 
- if you do have the ability to test on another network using the same device, does the issue occur on another network as well?

**Reply by smeijer** [reply]
I've just tested it in your app on different machines and networks, and it seems to work just fine here.

Can you please:

- try it in a different browser
- without any extensions
- in incognito

I'm suspecting either a proxy or browser extension that is dropping headers.

**Reply by TD-4242** [reply]
- is this on a corporate/public network, or at home?
Home and company network with standard firewall
- any chance that a proxy is removing headers?
no proxy in place
- do you have the same issue on another device?
yes, first found on Linux/Chrome, tested on Windows/Chrome and Android/Chrome
- if you do have the ability to test on another network using the same device, does the issue occur on another network as well?
Using VPN and trying different geographical locations the problem seems to be located in the western US
- try it in a different browser
Firefox same issue
- without any extensions
- in incognito
combined above, same issue

**Reply by TD-4242** [reply]
this is still happening in US Westcoast today.

**Reply by TD-4242** [reply]
Doesn't work:
- Phoenix
- Seattle
- Los Angeles
- San Francisco
- Denver

Works:
- New York
- Toronto
- Madrid
- Atlanta
- Dallas
- Miami
- Mexico

**Reply by TD-4242** [reply]
I just found it only happens in the latest two releases of the magicbell react-headless sdk.  I noticed one of our other apps was working fine so I reverted the version from 4.5.0 and 4.5.1 down to 4.0.3 and it seems to work fine in testing environment.

Although they were all working as we have been heavily testing the last two weeks up until our Monday release to all customers.

**Reply by TD-4242** [reply]
looks like the older version only supports the websocket protocol which is working.  it's seems only the sse protocol from ably that is being problematic.

**Comment by TD-4242** [comment]
after a lot of testing I've found this to be regionally effected.  Hana was able to log into our app and see realtime notifications but anybody here in the US was unable to.  I tried using a VPN to show my traffic from another location and it worked in some places like Canada and New York, but no where on the west coast of the US that I tried.

**Comment by TD-4242** [comment]
Any word back from Ably on this?  We are still seeing regional issues.

**Comment by unamashana** [comment, marked as answer]
We were able to work with Ably.com to resolve this issue and if you are seeing this, please feel free to create a new discussion. Thanks!
